h2o-py/tests/testdir_algos/rulefit/pyunit_titanic_rulefit.py

```diff
@@ -17,6 +17,8 @@ def titanic():
     rfit.train(training_frame=train, x=x, y="survived")
 
     print(rfit.rule_importance())
+    assert rfit._model_json["output"]["model_summary"] is not None
```

Comment:
Assert with a message would be better. :)

Reply:
49d3cc2